src/components/Editor/index.js

```diff
@@ -73,7 +77,9 @@ const Editor = React.createClass({
         caseSensitive: true,
         wholeWord: false,
         regexMatch: false
-      }
+      },
+      functionSearchEnabled: false,
+      functionDeclarations: null
```

Comment:
this should be a different name. Also we keep it here for memoization, but it should probably be kept in the util or redux store so that it is available if you change documents.